storage/etcd/config.go

```diff
@@ -0,0 +1,82 @@
+package etcd
+
+import (
+	"time"
+
+	"github.com/coreos/dex/storage"
+	"github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/pkg/transport"
+	"github.com/sirupsen/logrus"
+)
+
+type EtcdSSL struct {
+	ServerName string `json:"server_name" yaml:"server_name"`
+	CAFile     string `json:"ca_file" yaml:"ca_file"`
+	KeyFile    string `json:"key_file" yaml:"key_file"`
+	CertFile   string `json:"cert_file" yaml:"cert_file"`
+}
+
+type Etcd struct {
```

Comment:
Allow a prefix

Reply:
Hmm what do you mean ? Like setting a etcd namespace ? I think its usually done now with the help of the grpc-proxy, isnt it ?

Reply:
> Like setting a etcd namespace ?

Yeah an equivalent of `etcd grpc-proxy start --namespace=<prefix>`

> I think its usually done now with the help of the grpc-proxy, isnt it ?

I think it depends on your setup. But yeah we can document the grpc-proxy solution then add native namespace functionality later.

Reply:
Still need struct tags

```
Endpoints []string `yaml:"endpoints"`
```